renovate.json

```diff
@@ -1,13 +1,9 @@
 {
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
     "extends": [
-      "config:base"
+      "config:base",
+      "schedule:earlyMondays"
     ],
     "includeForks": true,
-    "labels": ["dependencies"],
-    "packageRules": [
-      {
-        "updateTypes": ["minor", "patch", "pin", "digest"],
-        "automerge": true
```

Comment:
@rajadileepkolli As we don't have a concrete test suite to verify the version upgrade validity, I would recommend not to autoMerge for now.